scripts/run-rosetta.sh

```diff
@@ -55,45 +55,37 @@ if [[ "${jsii_pkgs_file}" = "" ]]; then
 fi
 
 rosetta_cache_file=$HOME/.s3buildcache/rosetta-cache.tabl.json
-rosetta_cache_opts=""
-genexample_cache_opts=""
-if [[ -f $rosetta_cache_file ]]; then
-    rosetta_cache_opts="--cache-from ${rosetta_cache_file}"
-    genexample_cache_opts="--cache-from ${rosetta_cache_file}"
+extract_opts=""
+if $infuse; then
+    extract_opts="--infuse"
 fi
 
 #----------------------------------------------------------------------
 
 # Compile examples with respect to "decdk" directory, as all packages will
 # be symlinked there so they can all be included.
 echo "💎 Extracting code samples" >&2
-$ROSETTA \
+time $ROSETTA extract \
     --compile \
     --verbose \
-    --output samples.tabl.json \
-    $rosetta_cache_opts \
+    --cache ${rosetta_cache_file} \
     --directory packages/decdk \
+    ${extract_opts} \
     $(cat $jsii_pkgs_file)
 
-
 if $infuse; then
-    echo "💎 Infusing examples back into assemblies" >&2
-    $ROSETTA infuse \
-        --verbose \
-        samples.tabl.json \
-        $(cat $jsii_pkgs_file)
-
     echo "💎 Generating synthetic examples for the remainder" >&2
-    time $scriptdir/../tools/@aws-cdk/generate-examples/bin/generate-examples \
-        $genexample_cache_opts \
-        --append-to samples.tabl.json \
+    time npx cdk-generate-synthetic-examples \
         $(cat $jsii_pkgs_file)
 
+    time $ROSETTA extract \
+        --compile \
+        --verbose \
```

Comment:
Did this not need a `--loose` to prevent failing on invalid synthetic examples?

Reply:
Perhaps that would be helpful but instead I made sure that generate-examples (currently) produces all valid examples.

Reply:
for extract, there is only a flag `--fail` that we do not set, so even with invalid examples we won't fail.